python-sdk/langwatch/src/langwatch/telemetry/span.py

```
@@ -0,0 +1,820 @@
+        # Skip self parameters because it doesn't really help with debugging, becomes just noise
+        if (
+            "self" in all_args
+            and len(all_args) > 0
+            and parameters[0].name == "self"
+        ):
+            self_ = all_args["self"]
+            if self.name is None:
+                try:
+                    self.update_name(f"{self_.__class__.__name__}.{func.__name__}")
+                except:
+                    pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)